sympy/polys/matrices/domainmatrix.py

```diff
@@ -57,13 +121,97 @@ def get_domain(cls, items_sympy, **kwargs):
         return K, items_K
 
     def convert_to(self, K):
+        r"""
+        Returns a DomainMatrix with the desired domain or field
+
+        Parameters
+        ==========
+
+        K : Represents the desired domain or field
+
+        Examples
+        =====
+
+        >>> from sympy import ZZ, CC
+        >>> from sympy.polys.matrices import DomainMatrix
+        >>> domainMatrix = DomainMatrix([
+        ...    [ZZ(1), ZZ(2)],
+        ...    [ZZ(3), ZZ(4)]], (2, 2), ZZ)
+
+        >>> domainMatrix.convert_to(CC)
+        DomainMatrix([[(1.0 + 0.0j), (2.0 + 0.0j)],
+        [(3.0 + 0.0j), (4.0 + 0.0j)]], (2, 2), CC)
```

Comment:
I think it's best to avoid using RR or CC in examples. This could be `ZZ_I` or otherwise `QQ`.